cmd/integration/integration.go

```diff
@@ -176,7 +181,7 @@ func startComponents(firstManifestURL, secondManifestURL, apiVersion string) (st
 	handler.delegate = m.Handler
 
 	// Scheduler
-	schedulerConfigFactory := factory.NewConfigFactory(cl)
+	schedulerConfigFactory := factory.NewConfigFactory(cl, util.NewTokenBucketRateLimiter(bindPodsQps, bindPodsBurst))
```

Comment:
Do we really need to rate-limit it in integration tests? Can't we just pass null here?